app/repositories/ArrivalMovementRepository.scala

```diff
@@ -557,18 +557,23 @@ class ArrivalMovementRepository @Inject()(
   private def dropLastUpdatedIndex(collection: JSONCollection): Future[Boolean] =
     collection.indexesManager.list.flatMap {
       indexes =>
-        if (indexes.exists(_.name.contains(oldLastUpdatedIndexName))) {
-
-          logger.warn(s"Dropping $oldLastUpdatedIndexName index")
-
-          collection.indexesManager
-            .drop(oldLastUpdatedIndexName)
-            .map(
-              _ => true
-            )
-        } else {
-          logger.info(s"$oldLastUpdatedIndexName does not exist or has already been dropped")
-          Future.successful(true)
+        val indexToDrop = indexes
+          .filter(_.name.contains(lastUpdatedIndexName))
+          .filter(x => x.expireAfterSeconds.map(_ != appConfig.cacheTtl).getOrElse(false))
+          .get(0)
```

Comment:
why get(0) rather than just .head?

Reply:
or headOption

Reply:
Yeah, me being daft. I'll switch it to headOption